packages/ckeditor5-page-break/src/pagebreakediting.js

```diff
@@ -66,16 +66,19 @@ export default class PageBreakEditing extends Plugin {
 			view: ( modelElement, { writer } ) => {
 				const label = t( 'Page break' );
 				const viewWrapper = writer.createContainerElement( 'div' );
-				const viewLabelElement = writer.createContainerElement( 'span' );
-				const innerText = writer.createText( t( 'Page break' ) );
+				const viewLabelElement = writer.createUIElement(
+					'span',
+					{ class: 'page-break__label' },
+					function( domDocument ) {
+						const domElement = this.toDomElement( domDocument );
+						domElement.innerText = t( 'Page break' );
+
+						return domElement;
+					}
+				);
 
 				writer.addClass( 'page-break', viewWrapper );
-				writer.setCustomProperty( 'pageBreak', true, viewWrapper );
```

Comment:
This should be preserved in the new code.

Reply:
It's alrady implemented by `toPageBreakWidget()`: https://github.com/ckeditor/ckeditor5/blob/b1d465c/packages/ckeditor5-page-break/src/pagebreakediting.js#L141.

Reply:
Oh, I missed it. So it was updated twice in the current version

Reply:
Haha, yup. I was so impressed with this "optimisation" that I forgot to test the original case. Thanks for pointing that out! :D